IdentityServer/v7/docs/content/tokens/refresh.md

```diff
@@ -47,36 +47,31 @@ var response = await client.RequestRefreshTokenAsync(new RefreshTokenRequest
 
 The [Duende.AccessTokenManagement](https://github.com/DuendeSoftware/Duende.AccessTokenManagement/wiki) library can be used to automate refresh & access token lifetime management in ASP.NET Core.
 
-## Refresh token security considerations
+## Binding refresh tokens
```

Comment:
Suggest we keep the heading named the same, as there are subsections not about binding refresh tokens
```suggestion
## Refresh token security considerations
```

Reply:
That also changes any URLs/bookmarks, right?

Reply:
Yeah - you can link to a section with a url fragment. I guess the page would at least still load, but still.

Reply:
Anyway, just mentioning as something to keep in mind in general.